.github/workflows/main.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     - name: Download Ktlint
       run: |
-        KTLINT="0.37.0"
+        KTLINT="0.40.0"
```

Comment:
As there are few errors with this version can you try 0.37.1

as it has a windows fix in that version

Reply:
reference - https://github.com/pinterest/ktlint/issues/761#issue-630042183

Reply:
@FareesHussain Thank you for the suggestion, I will confirm with 0.37.1 and update.

Reply:
resolving on behalf of @FareesHussain